evaluation/swe_bench/run_infer.py

```diff
@@ -316,7 +316,7 @@ def process_instance(
         'swe_instance': instance.to_dict(),  # SWE Bench specific
         'instruction': instruction,
         'git_patch': git_patch,  # SWE Bench specific
-        'metadata': metadata,
+        'metadata': metadata.model_dump(),
```

Comment:
```suggestion
        'metadata': metadata.model_dump(mode='json'),
```

I tested with mode='json' meanwhile and it looks ok, but without mode should be ok too. Explicit mode may be a little safer, because apparently the default mode may contain non-serializable contents, but our metadata doesn't, so it shouldn't matter.